services/attestationaggregator/standard/metrics.go

```diff
@@ -119,20 +141,25 @@ func (s *Service) setupAttestationAggregationMetrics() error {
 	return nil
 }
 
-// AttestationAggregationCompleted is called when an attestation aggregationprocess has completed.
-func (s *Service) AttestationAggregationCompleted(started time.Time, slot phase0.Slot, result string) {
+func monitorAttestationAggregationCompleted(started time.Time, slot phase0.Slot, result string, chainTime chaintime.Service) {
```

Comment:
I'm not a big fan of passing the service around in function calls.  It makes more sense to pass in either `startOfSlot` or `slotElapsed` as a variable here (and elsewhere this happens).